pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "npc-gzip"
+version = "0.1.0"
+description = "Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors"
+authors = ["Zach Bloss <zacharybloss@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "npc_gzip"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.25.1"
+tqdm = "^4.65.0"
+
+[tool.poetry.group.dev.dependencies]
+black = "^23.7.0"
+isort = "^5.12.0"
+torchtext = "^0.15.2"
+portalocker = "^2.7.0"
+scikit-learn = "^1.3.0"
+
+
```

Comment:
Since splitting the `dev` dependency group has added doubled newlines in other places, this is no longer the only one. Although consistency could still be improved (for example, by turning each occurrence of two blank lines into one), I consider this review comment no longer current.